src/algorithms/all_paths.c

```diff
@@ -12,6 +12,7 @@
 // Append given 'node' to given 'level' array.
 static void _AllPathsCtx_AddConnectionToLevel(AllPathsCtx *ctx, uint level, Node *node,
 											  Edge *edge) {
+	if(!node) return;
```

Comment:
Why did you add this line?

Reply:
allow this op to be part of optional match flow

Reply:
I think that the caller operation should not call `All_path` if `src` is NULL